framework/source/class/qx/ui/layout/Grid.js

```diff
@@ -1191,7 +1192,7 @@ qx.Class.define("qx.ui.layout.Grid",
         };
       }
 
-      if (this.__colSpans.length > 0) {
+      if (this.getColSpans().length > 0) {
```

Comment:
Missing an underscore? `_getColSpans` vs `getColSpans`?

Reply:
@derrell thank you! corrected!